src/opnsense/mvc/app/controllers/OPNsense/Base/ControllerBase.php

```diff
@@ -44,32 +44,15 @@ class ControllerBase extends ControllerRoot
      * @param OPNsense\Core\Config $cnf config handle
      * @return ViewTranslator
      */
-    public function getTranslator($cnf)
+    public function getTranslator()
     {
-        $lang = 'en_US';
-
-        foreach ($cnf->object()->system->children() as $key => $node) {
-            if ($key == 'language') {
-                $lang = $node->__toString();
-                break;
-            }
-        }
-
-        $lang_encoding = $lang . '.UTF-8';
-        $textdomain = 'OPNsense';
-
         $ret = new ViewTranslator(array(
             'directory' => '/usr/local/share/locale',
-            'defaultDomain' => $textdomain,
-            'locale' => $lang_encoding,
+            'defaultDomain' => 'OPNsense',
+            'locale' => "en_US.UTF-8",
```

Comment:
https://github.com/phalcon/cphalcon/blob/master/phalcon/translate/adapter/gettext.zep#L198
don't setting locale under OPNSense, because setlocale() return FALSE. We can pass any value of 'locale' to ViewTranslator(). The effective setting 'locale' is below.